src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -125,10 +128,50 @@ public Response isAdmin(@PathParam("instanceName") String instanceName) throws I
         return Response.ok(body).build();
     }
 
+    @GET
+    @Path("{instanceName}/admin/list")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getAdminLis(@PathParam("instanceName") String instanceName) throws IOException, ApiException, GeneralSecurityException {
+        UserProfile userProfile = UserProfileManager.getUserProfile();
+        String token = userProfile.getAccessToken();
+        GitHubClient client = new GitHubClient();
+        client.setOAuth2Token(token);
+
+        Kabanero instance = KabaneroClient.getAnInstance(instanceName);
+        if (instance == null) {
+            return Response.status(404).entity(new ResponseMessage(instanceName + " not found")).build();
+        }
+
+        String instanceGithubOrg = instance.getSpec().getGithub().getOrganization();
```

Comment:
There are specificed admin teams in the org (not all teams are admin teams) these team names are listed in the  `spec.github.teams` list. Only members in those teams are considered admins. 

I don't see where you use those specified admin teams.